CHANGELOG.md

```diff
@@ -19,10 +19,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added: New trick, Cross Bomb Launch.
 - Changed: The Shinesink Clip and Aim Down Clip tricks are now a single Floor Clip trick.
 
+##### Artaria
+
+- Added: Video: Ballsparking into Speed Hallway from the right, charging speed from Energy Recharge Station South.
+- Changed: Doors into Charge Beam Access are excluded from being randomized into Grapple Beam Doors, in order to fascilitate Ballsparking into Speed Hallway.
+
 ##### Burenia
 
-- Added: Crossing the gap in Underneath Drogyga with Cross Bomb Launch (Intermediate).
+- Added: Crossing the gap in Underneath Drogyga with Cross Bomb Launch; Intermediate with Spin Boost, Advanced without.
 - Added: Reaching the Missile Tank Pickup in Main Hub Tower Top using Cross Bomb Launch (Advanced).
+- Added: Video: Morph Ball Movement Jump in Main Hub Tower Middle.
 
 ##### Cataris
 
```

Comment:
The added Cataris video could probably get an entry

Reply:
The trick itself is being added this version, so I figured that I didn't need to mention that it has a video in the changelog entry. I've typically not mentioned in the changelog entry that a new logic piece has a video attached.